Includes/xbe.h

```diff
@@ -0,0 +1,25 @@
+#ifndef XBE_H
+#define XBE_H
+
+#define XBETYPESTRING 0x48454258
+#define XBENAMESIZE 0x028
+#define SECTOR_SIZE 0x1000
+#define BASEADDRESS 0x10000
+
+typedef struct _XBE{
+  uint32_t type;
+  uint8_t  padding0[0xFF];
+  uint32_t baseAddress;
+  uint32_t sizeOfHeaders;
```

Comment:
nit: As this struct is only to get these fields at the right offset, you should probably add a comment which documents the offsets of each field `uint32_t sizeOfHeaders; // +264`; this makes it easier to confirm correctness and shows the intention of all the padding.